src/com/alevel/module1/Main.java

```diff
@@ -0,0 +1,120 @@
+package com.alevel.module1;
+
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.Random;
+import java.util.Scanner;
+
+
+public class Main {
+    public static void main(String[] args) {
+        final Scanner in = new Scanner(System.in);
```

Comment:
Why create this variable?

Reply:
Because I am using this variable on line 20.